### PR TITLE
JMLL-1150 Update the ServiceNow ingest

### PR DESCRIPTION
**Pull request by SamDudley**
This commit will also:
* add .dockerignore
* add a pre-commit hook script which uses a local venv

### CODE REVIEW

File: core/utils/staff_index.py [hunk 1]
```diff
@@ -333,8 +327,9 @@ def consolidate_staff_documents(
             "email_addresses": staff_document.staff_sso_email_addresses or [],
             "contact_phone": staff_document.people_finder_phone or "",
             "grade": staff_document.people_finder_grade or "",
-            "department": staff_document.service_now_department_id or "",
-            "department_name": staff_document.service_now_department_name or "",
+            # TODO: staff_document no longer has the department id or name
+            "department": "",
+            "department_name": "",
```
Comment (jafacakes2011) [inline core/utils/staff_index.py:332]: I think it's fine to strip these out, if there is anywhere that references them then we can either get rid, or refer to the hardcoded department ID

Comment (jafacakes2011) [reply core/utils/staff_index.py:332]: `settings.SERVICE_NOW_DIT_DEPARTMENT_SYS_ID`

**Review by jafacakes2011 — APPROVED** [review]
Just a couple of comments regarding the TODO

File: core/utils/staff_index.py (hunk 1 above)
Comment (SamDudley) [reply core/utils/staff_index.py:332]: I've compromised as it's referred to in the staff_search app.

**Pull request merged**